tests/conformance/state/state.go

```diff
@@ -453,6 +453,100 @@ func ConformanceTests(t *testing.T, props map[string]string, statestore state.St
 				}
 			}
 		})
+
+		t.Run("transaction-order", func(t *testing.T) {
```

Comment:
Is there any sequence to this test? 

Is statestore expected to be initialized with some data set, i ask this because i see a `delete` on `key2` but not for `key1`

Reply:
Yes the tests are currently run in sequence ...

Reply:
This is a standalone test and the only prerequisite is `init`.

There is no expectation from any other tests to set value. That's why we have prerequisites to upsert and delete various keys as needed for the test.